spec/features/donation_spec.rb

```diff
@@ -0,0 +1,15 @@
+RSpec.feature "Donation management", type: :feature do
+  before do
+    sign_in(@user)
+  end
+  let!(:url_prefix) { "/#{@organization.to_param}" }
+
+  scenario "User must confirm extremely large donation quantities", js: true do
+    visit url_prefix + "/donations/new"
+    fill_in "Quantity", with: 1_000_000
+
+    accept_confirm do
+      click_button "Save"
+    end
```

Comment:
2 things:
 1. This should be added to `spec/system/donation_system_spec.rb` -- sorry for not communicating that; I constantly forget :man_facepalming: 
 2. What is the resolution to this action? Assuming that they have confirmed they're donating a million units, what is the outcome? Could you add an expectation to check for that? (cf. that system spec, line 248)

Reply:
Done! The spec includes a stray `expect(page).not_to have_xpath(...)` in order to force the next page to load.